rpm/stackdriver-agent.spec

```diff
@@ -269,6 +272,13 @@ rm -rf %{buildroot}%{_prefix}/include/curl %{buildroot}%{_prefix}/lib/libcurl*
 %{__rm} -f %{buildroot}%{_prefix}/man/man5/%{programprefix}collectd-snmp.5*
 %{__rm} -f %{buildroot}%{_prefix}/bin/stackdriver-utils_vl_lookup_test
 
+%if %{bundle_yajl}
+mkdir -p %{buildroot}%{_libdir}/yajl/
+cp /usr/lib64/libyajl.so.1 %{buildroot}%{_libdir}/yajl/
+ln -s %{buildroot}%{_libdir}/yajl/libyajl.so.1 %{_libdir}/libyajl.so.1
```

Comment:
```sh
cd %{buildroot}%{_libdir} && ln -s yajl/libyajl.so.1 .
```
You want that link to be relative, not absolute.

Reply:
Actually, even simpler:
```sh
ln -s -t %{buildroot}%{_libdir} yajl/libyajl.so.1
```
(no need for the subshell in that case).

Reply:
Done.